QEfficient/cloud/infer.py

```diff
@@ -115,7 +115,7 @@ def main(
     #############################################
     # hf model -> export -> compile -> execute
     #############################################
-    model_hf = AutoModelForCausalLM.from_pretrained(model_hf_path, use_cache=True)
+    model_hf = AutoModelForCausalLM.from_pretrained(model_hf_path, use_cache=True, trust_remote_code=True, attn_implementation="eager")
```

Comment:
Here also, we should either remove the `trust_remote_code=True` or make it depend on a flag.

Reply:
Done